test/models/test_real_world.py

```diff
@@ -63,37 +63,41 @@ def test(t, t2): return model(t, Tensor([801]), t2).realize()
     helper_test("test_sd", lambda: (Tensor.randn(1, 4, 64, 64),Tensor.randn(1, 77, params["ctx_dim"])), test, 18.0, 513 if CI else 839)
 
   def test_unet_resblock(self):
-    model = [ResBlock(16, 24, 16) for _ in range(4)]
-    derandomize_model(model)
-    @TinyJit
-    def test(t, t2):
-      for l in model: t = l(t, t2)
-      return t.realize()
-    helper_test("test_unet_resblock", lambda: (Tensor.empty(4, 16, 8, 8), Tensor.empty(1, 24)), test, 0.01, 43)
+    with Context(RUN_PROCESS_REPLAY = 2 if RUN_PROCESS_REPLAY else 0):
+      model = [ResBlock(16, 24, 16) for _ in range(4)]
+      derandomize_model(model)
+      @TinyJit
+      def test(t, t2):
+        for l in model: t = l(t, t2)
+        return t.realize()
+      helper_test("test_unet_resblock", lambda: (Tensor.empty(4, 16, 8, 8), Tensor.empty(1, 24)), test, 0.01, 43)
 
   @unittest.skipUnless(is_dtype_supported(dtypes.float16), "need dtypes.float16")
   def test_llama(self):
-    dtypes.default_float = dtypes.float16
+    with Context(RUN_PROCESS_REPLAY = 2 if RUN_PROCESS_REPLAY else 0):
```

Comment:
wanna add this to `def setUp` and `def tearDown`?
example: extra/optimization/test_beam_search.py